src/Conversion/InterventionImageFileConverter.php

```diff
@@ -0,0 +1,181 @@
+ */
+class InterventionImageFileConverter implements FileConverter
+{
+    public function supportsConversion(string $fromFormat, string $toFormat, array $options = []): bool
+    {
+        $unsupportedOptions = $this->validateOptions($options);
+        if (!empty($unsupportedOptions)) {
+            return false;
+        }
+        // This converter requires intervention image as the image backend
+        $backend = Injector::inst()->get(Image_Backend::class);
+        if (!is_a($backend, InterventionBackend::class)) {
+            return false;
+        }
+        return $this->supportedByIntervention($fromFormat, $backend) && $this->supportedByIntervention($toFormat, $backend);
+    }
+
+    public function convert(DBFile $from, string $toFormat, array $options = []): DBFile
+    {
+        // Do some basic validation up front for things we know aren't supported
+        $problems = $this->validateOptions($options);
+        if (!empty($problems)) {
+            throw new FileConversionException('Invalid options provided: ' . implode(', ', $problems));
+        }
+        $originalBackend = $from->getImageBackend();
+        if (!is_a($originalBackend, InterventionBackend::class)) {
+            $actualClass = $originalBackend ? get_class($originalBackend) : 'null';
+            throw new FileConversionException("ImageBackend must be an instance of InterventionBackend. Got $actualClass");
+        }
+        if (!$this->supportedByIntervention($toFormat, $originalBackend)) {
+            throw new FileConversionException("Convertion to format '$toFormat' is not suported.");
+        }
+
+        $quality = $options['quality'] ?? null;
+        // Clone the backend if we're changing quality to avoid affecting other manipulations to that original image
+        $backend = $quality === null ? $originalBackend : clone $originalBackend;
+        // Pass through to invervention image to do the conversion for us.
+        try {
+            $result = $from->manipulateExtension(
+                $toFormat,
+                function (AssetStore $store, string $filename, string $hash, string $variant) use ($backend, $quality) {
+                    if ($quality !== null) {
+                        $backend->setQuality($quality);
+                    }
+                    $config = ['conflict' => AssetStore::CONFLICT_USE_EXISTING];
+                    $tuple = $backend->writeToStore($store, $filename, $hash, $variant, $config);
+                    return [$tuple, $backend];
+                }
+            );
+        } catch (ImageException $e) {
+            throw new FileConversionException('Failed to convert: ' . $e->getMessage(), $e->getCode(), $e);
+        }
+        // This is very unlikely but the API for `manipulateExtension()` allows for it
+        if ($result === null) {
+            throw new FileConversionException('File conversion resulted in null. Check whether original file actually exists.');
+        }
```

Comment:
I can't find a way to test either of these - I think all of the scenarios that would normally result in these are being explicitly validated against already. This is just here as some defensive programming.